src/Arcus.BackgroundJobs.Tests.Integration/KeyVault/AutoInvalidateKeyVaultSecretJobTests.cs

```diff
@@ -1,43 +1,51 @@
 ﻿using System;
 using System.Threading.Tasks;
 using Arcus.BackgroundJobs.Tests.Integration.Hosting;
+using Arcus.BackgroundJobs.Tests.Integration.KeyVault.Fixture;
 using Arcus.Security.Core;
 using Arcus.Security.Core.Caching;
-using Arcus.Security.Providers.AzureKeyVault.Authentication;
 using Arcus.Testing.Logging;
-using Microsoft.Azure.KeyVault;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Moq;
 using Polly;
 using Xunit;
 using Xunit.Abstractions;
 
-namespace Arcus.BackgroundJobs.Tests.Integration.Jobs
+namespace Arcus.BackgroundJobs.Tests.Integration.KeyVault
 {
     [Trait(name: "Category", value: "Integration")]
-    public class AutoInvalidateKeyVaultSecretJobTests : IAsyncLifetime
+    public class AutoInvalidateKeyVaultSecretJobTests
     {
-        private readonly ITestOutputHelper _outputWriter;
+        private readonly ILogger _logger;
         private readonly TestConfig _config;
-        private readonly TestHost _host;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AutoInvalidateKeyVaultSecretJobTests"/> class.
         /// </summary>
         public AutoInvalidateKeyVaultSecretJobTests(ITestOutputHelper outputWriter)
         {
-            _outputWriter = outputWriter;
+            _logger = new XunitTestLogger(outputWriter);
             _config = TestConfig.Create();
-            _host = new TestHost(_config, ConfigureServices);
         }
 
-        private void ConfigureServices(IServiceCollection services)
+        [Fact]
+        public async Task NewSecretVersion_TriggersKeyVaultJob_AutoInvalidatesSecret()
```

Comment:
Great stuff! This test seams to work again, so I 'unskipped' it.
(Hehe, 'unskipped' seems like a 1984 restricted word 😄 ).